docs/configuration/authentication/oidc_machtigen.rst

```diff
@@ -0,0 +1,100 @@
+.. _configuration_authentication_oidc_digid_machtigen:
+
+=============================================================================
+OpenID Connect voor inloggen met DigiD Machtigen en eHerkenning bewindvoering
+=============================================================================
+
+Open Formulieren ondersteunt `DigiD Machtigen`_ en eHerkenning bewindvoering login voor burgers via het OpenID Connect
+protocol (OIDC).
+Burgers kunnen inloggen op Open Formulieren met hun DigiD/eHerkenning account en een formulier invullen namens iemand
+anders. In deze flow:
+
+* Een gebruiker klikt op de knop *Inloggen met DigiD Machtigen* of *Inloggen met eHerkenning bewindvoering* die op de startpagina van een formulier staat.
+* De gebruiker wordt via de omgeving van de OpenID Connect provider (bijv. `Keycloak`_) naar DigiD/eHerkenning geleid, waar de gebruiker kan inloggen met *zijn/haar eigen* DigiD/eHerkenning inlog gegevens.
```

Comment:
```suggestion
* De gebruiker wordt via de omgeving van de OpenID Connect provider (bijv. `Keycloak`_) naar DigiD/eHerkenning geleid, waar de gebruiker kan inloggen met hun eigen* DigiD/eHerkenning inloggegevens.
```
bit more gender neutral language

Reply:
A nice! I didn't know it also worked in dutch! :tada: